src/character.cpp

```diff
@@ -2305,7 +2320,7 @@ hp_part Character::body_window( const std::string &menu_header,
     if( !is_valid_choice ) { // no body part can be chosen for this item
         bmenu.init();
         bmenu.desc_enabled = false;
-        bmenu.text = string_format( _( "No healable part for: %s" ), menu_header );
+        bmenu.text = string_format( _( "No limb would benefit from using a %s." ), menu_header );
```

Comment:
It may not be applicable for now, but nothing is preventing some further usage calls this function with the NPC name and `show_all = false`.

Reply:
I'll look into it when possible, but I thought I got it covered already somehow.

Reply:
Done. Also generalized other texts, so they don't refer to "item" explicitly, as in future it might be some kind of other effect (stationary medical equipment, autodoc, etc.)